`search_trigger: off` does not disable the default `alt+space` search trigger

### Description

According to the [docs](https://espanso.org/docs/configuration/options/#customizing-the-search-trigger), having the key value pair `search_trigger: off` in `$CONFIG/config/default.yml` should disable the search bar trigger completely. It does not.

### Steps to reproduce

1. Add `search_trigger: off` to $CONFIG/config/default.yml`
2. Reload config
3. Press `alt+space` 
4. Search trigger appears

### Expected behavior

Nothing should happen

### Screenshots

_No response_

### Logs

```
11:43:41 [daemon(31710)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:43:41 [daemon(31710)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:43:41 [daemon(31710)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:43:41 [daemon(31710)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:43:41 [daemon(31710)] [INFO] watching for changes in path: "/Users/nbbaier/Library/Application Support/espanso"
11:43:41 [daemon(31710)] [INFO] espanso version: 2.2.1
11:43:41 [daemon(31710)] [INFO] spawning the worker process...
11:43:41 [daemon(31710)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansodaemonv2.sock
11:43:41 [worker(31711)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:43:41 [worker(31711)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:43:41 [worker(31711)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:43:41 [worker(31711)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:43:41 [worker(31711)] [INFO] using CocoaAppInfoProvider
11:43:41 [worker(31711)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:43:41 [worker(31711)] [INFO] monitoring the status of the daemon process
11:43:41 [worker(31711)] [INFO] monitoring the status of secure input
11:43:41 [worker(31711)] [INFO] using CocoaSource
11:43:41 [worker(31711)] [INFO] using MacInjector
11:43:41 [worker(31711)] [INFO] using CocoaClipboard
11:43:43 [worker(31711)] [INFO] engine eventloop has terminated, propagating exit event...
11:43:43 [worker(31711)] [INFO] waiting for engine exit mode...
11:43:43 [worker(31711)] [INFO] exiting worker (to be restarted)
11:43:43 [daemon(31710)] [INFO] worker requested a restart, spawning a new one...
11:43:43 [daemon(31710)] [INFO] spawning the worker process...
11:43:43 [worker(31712)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:43:43 [worker(31712)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:43:43 [worker(31712)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:43:43 [worker(31712)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:43:43 [worker(31712)] [INFO] using CocoaAppInfoProvider
11:43:43 [worker(31712)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:43:43 [worker(31712)] [INFO] monitoring the status of secure input
11:43:43 [worker(31712)] [INFO] using CocoaSource
11:43:43 [worker(31712)] [INFO] monitoring the status of the daemon process
11:43:43 [worker(31712)] [INFO] using MacInjector
11:43:43 [worker(31712)] [INFO] using CocoaClipboard
11:43:45 [worker(31712)] [ERROR] modulo reported an error: 2025-03-20 11:43:44.777 espanso[31714:1002510] +[IMKClient subclass]: chose IMKClient_Modern

11:43:53 [daemon(31710)] [INFO] configuration change detected, restarting worker process...
11:43:53 [worker(31712)] [INFO] engine eventloop has terminated, propagating exit event...
11:43:53 [worker(31712)] [INFO] waiting for engine exit mode...
11:43:53 [worker(31712)] [INFO] exiting worker process...
11:43:53 [daemon(31710)] [INFO] spawning the worker process...
11:43:53 [worker(31718)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:43:53 [worker(31718)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:43:53 [worker(31718)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:43:53 [worker(31718)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:43:53 [worker(31718)] [INFO] using CocoaAppInfoProvider
11:43:53 [worker(31718)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:43:53 [worker(31718)] [INFO] monitoring the status of secure input
11:43:53 [worker(31718)] [INFO] using CocoaSource
11:43:53 [worker(31718)] [INFO] monitoring the status of the daemon process
11:43:53 [worker(31718)] [INFO] using MacInjector
11:43:53 [worker(31718)] [INFO] using CocoaClipboard
11:43:55 [daemon(31710)] [INFO] configuration change detected, restarting worker process...
11:43:55 [worker(31718)] [INFO] engine eventloop has terminated, propagating exit event...
11:43:55 [worker(31718)] [INFO] waiting for engine exit mode...
11:43:55 [worker(31718)] [INFO] exiting worker process...
11:43:55 [daemon(31710)] [INFO] spawning the worker process...
11:43:55 [worker(31720)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:43:55 [worker(31720)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:43:55 [worker(31720)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:43:55 [worker(31720)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:43:55 [worker(31720)] [INFO] using CocoaAppInfoProvider
11:43:55 [worker(31720)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:43:55 [worker(31720)] [INFO] monitoring the status of the daemon process
11:43:55 [worker(31720)] [INFO] monitoring the status of secure input
11:43:55 [worker(31720)] [INFO] using CocoaSource
11:43:55 [worker(31720)] [INFO] using MacInjector
11:43:55 [worker(31720)] [INFO] using CocoaClipboard
11:43:57 [worker(31720)] [ERROR] modulo reported an error: 2025-03-20 11:43:56.586 espanso[31721:1002899] +[IMKClient subclass]: chose IMKClient_Modern

11:43:58 [worker(31720)] [ERROR] modulo reported an error: 2025-03-20 11:43:58.059 espanso[31722:1002938] +[IMKClient subclass]: chose IMKClient_Modern

11:46:41 [daemon(31710)] [INFO] configuration change detected, restarting worker process...
11:46:41 [worker(31720)] [INFO] engine eventloop has terminated, propagating exit event...
11:46:41 [worker(31720)] [INFO] waiting for engine exit mode...
11:46:42 [worker(31720)] [INFO] exiting worker process...
11:46:42 [daemon(31710)] [INFO] spawning the worker process...
11:46:42 [worker(31754)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:46:42 [worker(31754)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:46:42 [worker(31754)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:46:42 [worker(31754)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:46:42 [worker(31754)] [INFO] using CocoaAppInfoProvider
11:46:42 [worker(31754)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:46:42 [worker(31754)] [INFO] monitoring the status of the daemon process
11:46:42 [worker(31754)] [INFO] using CocoaSource
11:46:42 [worker(31754)] [INFO] monitoring the status of secure input
11:46:42 [worker(31754)] [INFO] using MacInjector
11:46:42 [worker(31754)] [INFO] using CocoaClipboard
11:46:44 [worker(31754)] [ERROR] modulo reported an error: 2025-03-20 11:46:44.209 espanso[31758:1005287] +[IMKClient subclass]: chose IMKClient_Modern

11:47:01 [worker(31754)] [WARN] detected inconsistent modifier state for keys [(Meta, 55)], sending compensating events...
11:47:04 [worker(31754)] [WARN] detected inconsistent modifier state for keys [(Meta, 55)], sending compensating events...
11:48:13 [worker(31754)] [ERROR] modulo reported an error: 2025-03-20 11:48:12.750 espanso[31892:1007382] +[IMKClient subclass]: chose IMKClient_Modern

11:49:15 [worker(31754)] [WARN] detected inconsistent modifier state for keys [(Alt, 58)], sending compensating events...
11:50:37 [worker(31754)] [WARN] detected inconsistent modifier state for keys [(Meta, 55)], sending compensating events...
11:51:34 [worker(31754)] [INFO] engine eventloop has terminated, propagating exit event...
11:51:34 [worker(31754)] [INFO] waiting for engine exit mode...
11:51:34 [worker(31754)] [INFO] exiting worker (to be restarted)
11:51:34 [daemon(31710)] [INFO] worker requested a restart, spawning a new one...
11:51:34 [daemon(31710)] [INFO] spawning the worker process...
11:51:34 [worker(31956)] [INFO] reading configs from: "/Users/nbbaier/Library/Application Support/espanso"
11:51:34 [worker(31956)] [INFO] reading packages from: "/Users/nbbaier/Library/Application Support/espanso/match/packages"
11:51:34 [worker(31956)] [INFO] using runtime dir: "/Users/nbbaier/Library/Caches/espanso"
11:51:34 [worker(31956)] [INFO] system info: Darwin v15.3.2 - kernel: 24.3.0
11:51:34 [worker(31956)] [INFO] using CocoaAppInfoProvider
11:51:34 [worker(31956)] [INFO] binded to IPC unix socket: /Users/nbbaier/Library/Caches/espanso/espansoworkerv2.sock
11:51:34 [worker(31956)] [INFO] monitoring the status of the daemon process
11:51:34 [worker(31956)] [INFO] monitoring the status of secure input
11:51:34 [worker(31956)] [INFO] using CocoaSource
11:51:34 [worker(31956)] [INFO] using MacInjector
11:51:34 [worker(31956)] [INFO] using CocoaClipboard
11:51:36 [worker(31956)] [ERROR] modulo reported an error: 2025-03-20 11:51:35.839 espanso[31957:1010757] +[IMKClient subclass]: chose IMKClient_Modern
```

### Your environment

- Public source code:
- Environment name and version `rustc 1.85.1`
- Operating system and version `Darwin Nicos-MacBook-Air.local 24.3.0 Darwin Kernel Version 24.3.0: Thu Jan  2 20:24:06 PST 2025; root:xnu-11215.81.4~3/RELEASE_ARM64_T8103 arm64`
- Espanso version `espanso 2.2.1`


### Self-service

- [ ] I'd be willing to fix this bug myself.

--- a/docs/configuration/options.md
+++ b/docs/configuration/options.md
@@ -3,28 +3,19 @@ sidebar_position: 4
 title: Options
 ---
 
-In this guide, we are going to discuss the available options to customize Espanso.
-The first few sections are about the common parameters you might want to change,
-but you'll also find a complete reference at the bottom for the advanced ones.
+In this guide, we are going to discuss the available options to customize Espanso. The first few sections are about the common parameters you might want to change, but you'll also find a complete reference at the bottom for the advanced ones.
 
 ## Customizing the Toggle Key
 
-If you often need to **quickly enable and disable Espanso** during regular use, you might want to customize
-the _Toggle Key_.
-When double pressed, the Toggle Key disables Espanso, preventing any expansion.
-Double-pressing Alt again will re-enable it. 
+If you often need to **quickly enable and disable Espanso** during regular use, you might want to customize the _Toggle Key_. When double pressed, the Toggle Key disables Espanso, preventing any expansion. Double-pressing Alt again will re-enable it. 
 
 :::info Changes in version 2.1.2
 
-Prior to version 2.1.2, Espanso was configured to use the ALT key as `toggle_key` by default.
-That was a major source of confusion, as many users accidentally pressed it during
-normal use.
-For this reason, Espanso now ships with the `toggle_key` _disabled_ by default.
+Prior to version 2.1.2, Espanso was configured to use the ALT key as `toggle_key` by default. That was a major source of confusion, as many users accidentally pressed it during normal use. For this reason, Espanso now ships with the `toggle_key` _disabled_ by default.
 
 :::
 
-If you'd like to customize the key, simply add the `toggle_key` parameter to your
-`$CONFIG/config/default.yml` configuration and set it to one of the available options:
+If you'd like to customize the key, simply add the `toggle_key` parameter to your `$CONFIG/config/default.yml` configuration and set it to one of the available options:
 
 |              |             |               |              |
 |--------------|-------------|---------------|--------------|------|
@@ -32,8 +23,7 @@ If you'd like to customize the key, simply add the `toggle_key` parameter to you
 | `LEFT_CTRL`  | `LEFT_ALT`  | `LEFT_SHIFT`  | `LEFT_META`  |
 | `RIGHT_CTRL` | `RIGHT_ALT` | `RIGHT_SHIFT` | `RIGHT_META` |
 
-For example, if you want to use the Right Control key to toggle Espanso, you can add the
-following to your configuration:
+For example, if you want to use the Right Control key to toggle Espanso, you can add the following to your configuration:
 
 ```yaml title="$CONFIG/config/default.yml"
 toggle_key: RIGHT_CTRL
@@ -41,12 +31,9 @@ toggle_key: RIGHT_CTRL
 
 :::tip About the META key
 
-The `META` option refers to different keys depending on the platform.
-For example, on macOS the `META` refers to Command (or CMD),
-while on Windows refers to the "Win" key.
+The `META` option refers to different keys depending on the platform. For example, on macOS the `META` refers to Command (or CMD), while on Windows refers to the "Win" key.
 
-In other words, if you'd like to use the CMD key on macOS, you should
-specify `META`.
+In other words, if you'd like to use the CMD key on macOS, you should specify `META`.
 
 :::
 
@@ -58,8 +45,7 @@ toggle_key: OFF
 
 ## Customizing the Search bar
 
-The Search bar is one of the most useful features, letting you choose the right snippet 
-without having to remember the trigger.
+The Search bar is one of the most useful features, letting you choose the right snippet without having to remember the trigger.
 
 By default, the search bar can be opened in two ways:
 * By pressing ALT+Space (Option + Space on macOS)
@@ -69,22 +55,19 @@ If for whatever reason you don't like the default shortcuts, you can customize t
 
 ### Customizing the search shortcut
 
-If you don't like the default Alt+Space shortcut, you can change it by adding the following line
-to your `$CONFIG/config/default.yml` file:
+If you don't like the default Alt+Space shortcut, you can change it by adding the following line to your `$CONFIG/config/default.yml` file:
 
 ```yaml title="$CONFIG/config/default.yml"
 search_shortcut: ALT+SHIFT+SPACE
 ```
 
-As you can see, the shortcut is defined by specifying all keys separated by a plus sign.
-These are the supported keys:
+As you can see, the shortcut is defined by specifying all keys separated by a plus sign. These are the supported keys:
 
 `ALT`, `CTRL`, `CMD`, `SHIFT`, `ENTER`, `TAB`, `SPACE`, `META`, `OPTION`, `INSERT`, `DOWN`, `LEFT`, `RIGHT`, `UP`, `END`, `HOME`, `PAGEDOWN`, `PAGEUP`, `F1`, `F2`, `F3`, `F4`, `F5`, `F6`, `F7`, `F8`, `F9`, `F10`, `F11`, `F12`, `F13`, `F14`, `F15`, `F16`, `F17`, `F18`, `F19`, `F20`, `A`, `B`, `C`, `D`, `E`, `F`, `G`, `H`, `I`, `J`, `K`, `L`, `M`, `N`, `O`, `P`, `Q`, `R`, `S`, `T`, `U`, `V`, `W`, `X`, `Y`, `Z`, `0`, `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9`, `NUMPAD0`, `NUMPAD1`, `NUMPAD2`, `NUMPAD3`, `NUMPAD4`, `NUMPAD5`, `NUMPAD6`, `NUMPAD7`, `NUMPAD8`, `NUMPAD9`
 
-> Not all keyboard combinations have been tested. If you find a combination that's not working, please open
-> a bug report on [GitHub](https://github.com/espanso/espanso/issues). Thanks!
+> Not all keyboard combinations have been tested. If you find a combination that's not working, please open a bug report on [GitHub](https://github.com/espanso/espanso/issues). Thanks!
 
-You can also disable this shortcut entirely by adding the following line:
+You can also disable the keybinding by adding the following line:
 
 ```yaml title="$CONFIG/config/default.yml"
 search_shortcut: off
@@ -92,15 +75,13 @@ search_shortcut: off
 
 ### Customizing the search trigger
 
-The search bar can also be configured to be shown after typing a trigger.
-For example, you might tune it to be displayed after typing `.search` by
-adding this option:
+The search bar can also be configured to be shown after typing a trigger. For example, you might set it to be displayed after typing `.search` by adding this option:
 
 ```yaml title="$CONFIG/config/default.yml"
 search_trigger: ".search"
 ```
 
-Or disable it entirely with:
+Disable the trigger by omitting the entry, or replacing it with:
 
 ```yaml title="$CONFIG/config/default.yml"
 search_trigger: off
@@ -108,8 +89,7 @@ search_trigger: off
 
 ## Hiding the Icon
 
-You can hide the Espanso status icon on the macOS menu bar or the Windows system tray by adding the following option to 
-your `$CONFIG/config/default.yml` file:
+You can hide the Espanso status icon on the macOS menu bar or the Windows system tray by adding the following option to your `$CONFIG/config/default.yml` file:
 
 ```yaml title="$CONFIG/config/default.yml"
 show_icon: false
@@ -125,8 +105,7 @@ show_notifications: false
 
 ## Options reference
 
-Here's a list of all the available options. If the value of the `App-specific` column is `Yes`, then the option
-can be used inside an [App-specific configuration](../app-specific-configurations).
+Here's a list of all the available options. If the value of the `App-specific` column is `Yes`, then the option can be used inside an [App-specific configuration](../app-specific-configurations).
 Otherwise, the option can only be defined inside the `$CONFIG/config/default.yml` file.
 
 Option 	 | 	 Description 	 | 	 Possible Values 	 | 	 Default 	 | 	 App-Specific 

